build-java/build.gradle

```diff
@@ -0,0 +1,27 @@
+buildscript {
+    repositories {
+        mavenLocal()
+        mavenCentral()
+    }
+
+    dependencies {
+        classpath "org.openapitools:openapi-generator-gradle-plugin:4.0.0-beta2"
+    }
+}
+
+apply plugin: 'org.openapi.generator'
+
+def clientOutputDir = "$projectDir/../generated-clients/java-clients".toString()
+
+task generateClient(type: GradleBuild) {
```

Comment:
What happens if you list all the tasks at the top dir?
Will you see twice "generateClient" task, since there is one for Python and one for Java?